lib/bots/versionbot.ts

```diff
@@ -414,6 +414,7 @@ export class VersionBot extends ProcBot {
         const head = event.cookedEvent.data.pull_request.head;
         const owner = head.repo.owner.login;
         const name = head.repo.name;
+        const author = event.cookedEvent.data.sender.login;
```

Comment:
Fixed to be author and committer (if the two are the same, then it only does one ping to the author).